cli/jobs/automl-standalone-jobs/cli-automl-forecasting-orange-juice-sales/test-mltable-folder/MLTable

```diff
@@ -0,0 +1,6 @@
+paths:
+- file: ./oj-sales-test.csv
+transformations:
+- read_delimited:
+    delimiter: ','
+    encoding: ascii
```

Comment:
I may be wrong, but I thought we need to use the convert_column_types parameter? This is what is being shown in the current v2 version (not created by our team) of the forecast [notebook](https://github.com/Azure/azureml-examples/blob/main/sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-energy-demand/data/test-mltable-folder/MLTable).

Reply:
@vlbejan
Nikolay confirmed that the convert_column_types is not required.